javamelody-core/src/main/java/net/bull/javamelody/MonitoringFilter.java

```diff
@@ -202,7 +202,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 		final HttpServletRequest httpRequest = (HttpServletRequest) request;
 		final HttpServletResponse httpResponse = (HttpServletResponse) response;
 
-		if (httpRequest.getRequestURI().equals(getMonitoringUrl(httpRequest))) {
+		if (httpRequest.getRequestURI().endsWith(getMonitoringUrl(httpRequest))) {

```

Comment:
This is a bit dangerous, as you may match *anything* that is ending with /monitoring.
Example /myapp/roadmap/monitoring ... will end up in JavaMelody instead of displaying (example) the monitoring stations of a road-map :-)

Reply:
Alternative approach: Allow to optionally define a list of supported monitoring URLs to check against. For gerrit we would  define: ["monitoring", "a/monitoring"].